frontend/app/src/components/history/LocationIcon.vue

```diff
@@ -1,7 +1,7 @@
 <script setup lang="ts">
 import { type PropType } from 'vue';
 import AdaptiveWrapper from '@/components/display/AdaptiveWrapper.vue';
-import { type TradeLocationData } from '@/types/trades';
+import { type TradeLocationData } from '@/types/history/trade/trade-location';
```

Comment:
nitpick: rename to trade-location.ts to location since it is already under trade